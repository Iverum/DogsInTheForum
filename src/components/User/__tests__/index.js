import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import { User } from '../index'

jest.mock('firebase')

test('user detail is rendered correctly', () => {
  const component = renderer.create(
    <User user={{ name: 'Test User' }}/>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
