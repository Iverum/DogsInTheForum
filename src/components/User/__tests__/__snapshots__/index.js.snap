exports[`test user detail is rendered correctly 1`] = `
<div
  className="twelve columns">
  <h1>
    Test User
  </h1>
</div>
`;
